tests/test_staging/test_staging_helpers.py

```diff
@@ -446,18 +446,30 @@ def create_exp_output_df(self):
             [49911791786, 1, "4", "201", "CF10 BZZ"],
             [49901183959, 4, "1", "309", "SA50 5BE"],
         ]
-        exp_output_df = pandasDF(data=data, columns=exp_output_columns)
-        exp_output_df["legalstatus"].astype("category")
-        exp_output_df["statusencoded"].astype("category")
-        return exp_output_df
+        exp1_output_df = pandasDF(data=data1, columns=exp_output_columns)
+        exp1_output_df["legalstatus"].astype("category")
+        exp1_output_df["statusencoded"].astype("category")
+
+        data2 = [
+            [49900000510, 2.0, "7", "201", "BA1 5DA"],
+            [49900184433, 1.0, "7", "210", "CF10 BZZ"],
+        ]
+        exp2_output_df = pandasDF(data=data2, columns=exp_output_columns)
+        exp2_output_df["legalstatus"].astype("category")
+        exp2_output_df["statusencoded"].astype("category")
+
+        return exp1_output_df, exp2_output_df
 
     def test_filter_pnp_data(self):
         """Test for the filter_pnp_data function."""
         input_df = self.create_input_df()
-        exp_df = self.create_exp_output_df()
+        exp1_df, exp2_df = self.create_exp_output_df()
 
-        result_df = filter_pnp_data(input_df)
+        result1_df, result2_df = filter_pnp_data(input_df)
 
         pd.testing.assert_frame_equal(
-            result_df.reset_index(drop=True), exp_df.reset_index(drop=True)
+            result1_df.reset_index(drop=True), exp1_df.reset_index(drop=True)
+        )
+        pd.testing.assert_frame_equal(
```

Comment:
Can you rename the class? The convention is to use CamelCase for the class, so it would be TestFilterPnpData. We use snake_case for function names.